### ISSUE TITLE
The bar format is saved between the instances

### ISSUE THREAD

**Issue by rvalitov** [AndiDittrich/Node.CLI-Progress#27]
Hi! If I use a progress bar with custom format defined, for example:

```js
const bar = new _cliProgress.Bar({
                format: 'Downloading [{bar}] {percentage}% | ETA: {eta}s | {value}/{total}',
            }, _cliProgress.Presets.shades_classic);
```
Then I see a bar with `Downloading` prefix, as expected. Later, I create a new bar with default options, i.e.:

```js
const bar2 = new _cliProgress.Bar({}, _cliProgress.Presets.shades_classic);
```
I expect to have a progress bar without any prefix, i.e. the default output format:

```
[{bar}] {percentage}% | ETA: {eta}s | {value}/{total}
```
But in this case I also see the `Downloading` prefix. To solve this issue, I always have to define the format, otherwise a new object will use the format of the previous object. I think that's not a correct behavior.

**Comment by AndiDittrich** [comment]
Hi @rvalitov ,

thanks for your report.

of course..it's a bug within the preset processing: the options are [overriding the preset object properties](https://github.com/AndiDittrich/Node.CLI-Progress/blob/master/lib/Bar.js#L13). i will fix it asap.

best regards, Andi

**Comment by AndiDittrich** [comment]
should be fixed in 4b5f656 by using `Object.assign`

**Comment by rvalitov** [comment]
Thank you! Please, don't forget to publish the new version, so that it's easier to update with npm.
